BBangmap/BBangmap/Global/DesignSystem/View/BottomSheetView.swift

```diff
@@ -5,4 +5,179 @@
+        $0.setBackgroundColor(Color.bbangWhite, for: .normal)
+        $0.setBackgroundColor(Color.bbangLightgray, for: .highlighted)
+
+        $0.titleLabel?.font = NotoSansKR.regular.of(size: 17)
+    }
+
+    private let cancelButton = UIButton().then {
+        $0.setTitle("취소", for: .normal)
+        $0.setTitleColor(Color.bbangBlack, for: .normal)
+        $0.setBackgroundColor(Color.bbangWhite, for: .normal)
+        $0.setBackgroundColor(Color.bbangLightgray, for: .highlighted)
+
+        $0.titleLabel?.font = NotoSansKR.regular.of(size: 17)
+
+        $0.cornerRadius = 20
+    }
+
+    private let publisher: PublishSubject<SheetButtonType>
+
+    init(
+        viewType: ViewType,
+        dismissPublisher: PublishSubject<SheetButtonType>
+    ) {
+        self.publisher = dismissPublisher
+        super.init()
+
+        render()
+        bindViewDismiss()
+        setup(viewType: viewType)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func render() {
+        view.backgroundColor = .clear
+        view.add(backgroundView)
+        buttonStackView.add(spliteView)
+        backgroundView.adds([buttonStackView, cancelButton])
+
+        backgroundView.snp.makeConstraints {
+            $0.edges.equalToSuperview()
+        }
+
+        buttonStackView.snp.makeConstraints {
+            $0.top.equalToSuperview().offset(616)
+            $0.leading.trailing.equalToSuperview().inset(16)
+        }
+
+        spliteView.snp.makeConstraints {
+            $0.leading.trailing.equalToSuperview()
+            $0.centerY.equalToSuperview()
+            $0.height.equalTo(0.5)
+        }
+
+        cancelButton.snp.makeConstraints {
+            $0.top.equalTo(buttonStackView.snp.bottom).offset(10)
+            $0.leading.trailing.equalToSuperview().inset(16)
+            $0.bottom.equalToSuperview().inset(36)
+        }
+    }
+
+    private func bindViewDismiss() {
+        Observable.merge(
+            backgroundView.rx.tapGesture().when(.recognized).map { _ in },
+            cancelButton.rx.tap.map { _ in }
+        )
+        .subscribe(onNext: { [weak self] _ in
+            guard let self = self else { return }
+            self.dismiss(animated: true)
+        })
+        .disposed(by: disposeBag)
+
+        modifyButton.rx.tap
+            .withUnretained(self)
+            .subscribe(onNext: { view, _ in
+                view.dismiss(animated: true) {
+                    view.publisher.onNext(.cancel)
+                }
+            })
+            .disposed(by: disposeBag)
+
+        Observable.merge(
+            modifyButton.rx.tap.withUnretained(self),
+            deleteButton.rx.tap.withUnretained(self)
+        )
+        .subscribe(onNext: { [weak self] _ in
+            guard let self = self else { return }
+            self.dismiss(animated: true) {
+
+            }
+        })
+        .disposed(by: disposeBag)
```

Comment:
```suggestion
        Observable.merge(
            modifyButton.rx.tap,
            deleteButton.rx.tap
        )
        .withUnretained(self)
        .subscribe(onNext: { view, _ in
            view.dismiss(animated: true) {

            }
        })
        .disposed(by: disposeBag)
```
가 더 좋아보입니다 🙏 
`.withUnretained(self)`는 이렇게 사용하는거일겁니다 호호...

Reply:
아 맞아유.... 이거 근데 다음 테스크로 할거라 여기선 패스할게요

Reply:
근데 이거 안되는거 같은디;;;

Reply:
어떤게 안되나염

Reply:
`view`가 이미 이름을 차지하고 있어서 안된다면 변경해주세요~